### ISSUE TITLE
Recent pubs growth value of -100%

### ISSUE THREAD

**Issue by za158** [georgetown-cset/parat#458]
![image](https://github.com/georgetown-cset/parat/assets/58824955/018f02b0-d684-471e-9c4f-d5f1e7df475d)

**Comment by za158** [comment]
The issue being: how can the average change be -100% for more than one year? Or are years with 0 change getting dropped?

**Comment by jmelot** [comment]
Years with 0 in the denominator are getting dropped

**Comment by za158** [comment]
How about we display "n/a" with a tooltip indicating "Insufficient data to calculate"? @brianlove

**Comment by brianlove** [comment]
@za158 This "N/A"+tooltip is for _any_ growth column with a total of `-100`?

**Comment by jmelot** [comment]
Yeah I'm not sure we should do this for any such column. We could instead update the preprocessing code so that in cases where _any_ of the pct changes would have a 0 in the denominator, we return null, triggering the existing N/A on the front end

**Comment by za158** [comment]
Seems reasonable
